charts/webank/Chart.yaml

```diff
@@ -4,18 +4,18 @@ description: A Helm chart for Kubernetes
 
 type: application
 
-version: 1.0.1
+version: 1.0.2-rc
```

Comment:
Use the version `1.0.1`. When the pipeline builds, use version `1.0.2-rc.1` then.